vxdev/update-code.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+
+: "${VX_CONFIG_ROOT:="/vx/config"}"
+APP_TYPE=$(sudo cat "$VX_CONFIG_ROOT/machine-type")
+
+cd /vx/code/vxsuite-complete-system
+git checkout main > /dev/null
```

Comment:
Is there any chance we'd want to pull down a branch that isn't main for pre-merge testing?

Reply:
Nothing that happens at this point relies on staying on the branch with the vxdev scripts as those are copied into other places on the drive earlier on. For the eng team internally people will definitely want to test things on branches sometimes but they can just go into vxsuite and go to the branch they want on their own for that and not run this program. At least initially.